aws-lc-sys/builder/main.rs

```diff
@@ -571,6 +568,40 @@ fn invoke_external_bindgen(
         }
         return Err("External bindgen command failed.".to_string());
     }
+    let mut major_version: u32 = 0;
+    let mut minor_version: u32 = 0;
+    let mut patch_version: u32 = 0;
+    let bindgen_version = result.stdout.split(' ').skip(1).next();
```

Comment:
we're confident this format will remain stable?

Reply:
Yeah, I can't imaging them moving away from the [semver standard](https://semver.org/) for version numbering format.